app/helpers/quadicon_helper.rb

```diff
@@ -70,7 +70,7 @@ def self.settings_key(klass, layout)
       when "ems_physical_infra", "ems_cloud", "ems_network", "ems_container"
         layout.to_sym
       end
-    elsif klass.base_model.name != klass.name.demodulize
+    elsif klass.name.demodulize.starts_with?("Physical") && klass.base_model.name != klass.name.demodulize
```

Comment:
I agree with that if we remove later at https://github.com/ManageIQ/manageiq-ui-classic/pull/3954
There is a dependency in the core there. So the current PR can be approved faster.